.devcontainer/data-portal.yaml

```diff
@@ -0,0 +1,7 @@
+# development configuration
+
+host: 127.0.0.1
+port: 8080
+
+mass_url: /api/mass
+metldata_url: /api/metldata
```

Comment:
We need to set up editoconfig for these small issues of missing end newlines

Reply:
We will add pre-commit hooks for checking this and many other things.

Reply:
The new-lines at the end of files should be taken care of by prettier. We need to add the prettier plugin, a suitable config and a commit hook for prettier. This is part of GSI-1104.

Reply:
Well, then our two options for this issue are disabled then 😄